[quant][graphmode] Call _jit_pass_dedup_module_ueses in quantize_script

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #32379 [WIP][quant][graphmode] FoldConvBatchNorm2d support shared ClassTypes
* #32543 [WIP][quant][graphmode] ConvBn Folding works with bias=None
* #32380 [quant][graphmode][refactor] Better API for fold_convbn
* #32374 [refactor] Adding FoldConvBatchNorm2dHelper
* **#32303 [quant][graphmode] Call _jit_pass_dedup_module_ueses in quantize_script**
* #32302 [quant] Re-enable fold_convbn in quantize_script
* #32206 [quant] Re-enable test_nested that has different qconfig for shared ClassType
* #32205 [quant][graphmode] Support quantizing shared ClassType with different qconfigs

Summary:
att

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D19508029](https://our.internmc.facebook.com/intern/diff/D19508029)

jerryzh168 committed:
[quant][graphmode] Call _jit_pass_dedup_module_ueses in quantize_script

Summary:
att

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Call _jit_pass_dedup_module_ueses in quantize_script"


Summary:
att

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit b6f2f576:


* **1/2** failures introduced in this PR
* **1/2** recognized as flaky :snowflake:
    * Re-run these jobs?


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=b6f2f5767751d359bc0bd21a54ff245ac832957d).

### :snowflake: 1 failure recognized as flaky
The following build failures have been detected as flaky and **may not be your fault**:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4282336) pytorch_python_doc_push (1/1)
**Step:** "Doc Build and Push" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=87385440) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=87385440)) :snowflake:
<details>
<summary>
<code>$<3>$<3> 7b3fc97cdb6e: Pushing  6.019GB/6.019GB $<3>$<3> 7b3fc97cdb6e: Pushing  6.019GB/6.019GB toomanyrequests: Rate exceeded</code>
</summary>

```
$<3>$<3> eea1661bdba1: Preparing   
$<3>$<3> 5bb9fbc1099e: Preparing   
$<3>$<3> 0d9b5dad09df: Preparing   
$<3>$<3> a80eb84c4169: Preparing   
$<3>$<3> 85775ce071b3: Preparing   
$<3>$<3> bfd07f554dc2: Preparing   
$<3>$<3> 00c6fc364d3a: Preparing   
$<3>$<3> 77008e118980: Preparing   
$<3>$<3> 6cb741cb00b7: Preparing   
$<3>$<3> f36b28e4310d: Preparing   
7b3fc97cdb6e: Pushing  6.019GB/6.019GB $<3>$<3> 7b3fc97cdb6e: Pushing  6.019GB/6.019GB toomanyrequests: Rate exceeded 
```

</details>

### 1 failure *not* recognized by patterns:
| Job | Step | Status |
| --- | --- | --- |
| ![CircleCI](https://avatars0.githubusercontent.com/ml/7?s=12) <sup>pytorch_linux_xenial_py3_clang5_asan_build</sup> | <sup>Build</sup> | New in PR |


---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

jerryzh168 committed:
Update on "[quant][graphmode] Call _jit_pass_dedup_module_ueses in quantize_script"


Summary:
att

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Call _jit_pass_dedup_module_ueses in quantize_script"


Summary:
att

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

ZolotukhinM reviewed (approved):
👍

jerryzh168 committed:
Update on "[quant][graphmode] Call _jit_pass_dedup_module_ueses in quantize_script"


Summary:
att

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Call _jit_pass_dedup_module_ueses in quantize_script"


Summary:
att

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Call _jit_pass_dedup_module_ueses in quantize_script"


Summary:
att

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Call _jit_pass_dedup_module_ueses in quantize_script"


Summary:
att

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D19508029](https://our.internmc.facebook.com/intern/diff/D19508029)

[ghstack-poisoned]
